src/main/js/main_html5.js

```diff
@@ -757,8 +765,8 @@ require("../../../html5-common/js/utils/environment.js");
         currentTime = _video.currentTime;
       }
 
-      if (queuedInitialTime && (event.target.currentTime >= queuedInitialTime)) {
-        queuedInitialTime = 0;
+      if (initialTime.value > 0 && (event.target.currentTime >= initialTime.value)) {
+        initialTime.value = 0;
```

Comment:
the reason i don't is because i set it to true when the seeked event is raised.  The value is used to block playheads before the time is reached and we don't want to block any playhead after this point but we do want to block seeked events even if we get to this point so i leave reached as false